### PR TITLE
Significant digits REPL representation 

### PR DESCRIPTION
**Pull request by BeastyBlacksmith**
shows only significant digits of `Measurement` objects in the REPL.
Relates to https://github.com/JuliaPhysics/Measurements.jl/issues/5 as a proof of concept.

### CODE REVIEW

File: src/Measurements.jl [hunk 1]
```diff
@@ -101,6 +101,17 @@ measurement
 # Type representation
 Base.show(io::IO, measure::Measurement) =
```
Comment (giordano) [inline src/Measurements.jl]: I think you can remove this method (or the method you defined can use the signature of this one)

**Comment by giordano** [comment]
Thanks for addressing this, I was thinking about something like this.

Please, add some tests to the "Test representation" `@testset`.

**Comment by BeastyBlacksmith** [comment]
still need to implement it for `Complex`

**Comment by BeastyBlacksmith** [comment]
That should be all?

**Comment by giordano** [comment]
Merged, thanks!

**Pull request merged**